### ISSUE TITLE
SystemProperties missing in EventData inside received PartitionEvent

### ISSUE THREAD

**Issue by piotr-napadlek** [Azure/azure-sdk-for-java#10686]
Library version:
```
		<dependency>
			<groupId>com.azure</groupId>
			<artifactId>azure-messaging-eventhubs</artifactId>
			<version>5.0.3</version>
		</dependency>
```
Sample code:
```
        EventHubConsumerClient consumerClient = EventHubClientBuilder()
                .connectionString(connectionString)
                .consumerGroup(consumerGroup)
                .buildConsumerClient();
       IterableStream<PartitionEvent> events = consumerClient.receiveFromPartition("0", 100, EventPosition.earliest(), Duration.ofSeconds(5));
```

Expected: list of events with `data` field containing SystemProperties eg. offset, sequenceNumber etc.

Actual result: list events with `data` field, but system properties is an empty map. Application properties are ok.

**Comment by joshfree** [comment]
@srnagar can you please take a look?

/cc @conniey

**Comment by shubhambhattar** [comment]
@piotr-napadlek `offset` and `sequenceNumber` are still available as:

```
public class Consume {

    public static void main(String[] args) {

        EventHubConsumerClient consumerClient = new EventHubClientBuilder()
                .connectionString("")
                .consumerGroup("")
                .buildConsumerClient();

        IterableStream<PartitionEvent> events = consumerClient.receiveFromPartition("0", 100, EventPosition.earliest(), Duration.ofSeconds(5));
        events.stream()
                .forEach(partitionEvent -> {
                    System.out.println("Offset = " + partitionEvent.getData().getOffset());
                    System.out.println("SequenceNumber = " + partitionEvent.getData().getSequenceNumber());
                });
    }
}
```

**Comment by srnagar** [comment]
@piotr-napadlek Thanks for reporting this issue. It has been addressed as part of #11038 and will be available in the next Event Hubs release.
